src/main/java/org/broadinstitute/consent/http/service/DarCollectionService.java

```diff
@@ -414,11 +414,13 @@ public DarCollectionSummary updateCollectionToDraftStatus(DarCollection sourceCo
     return this.processDraftAsSummary(new ArrayList<>(sourceCollection.getDars().values()).get(0));
   }
 
-  public List<Integer> findDatasetIdsByUser(User user) {
-    return datasetDAO.findDatasetsByAuthUserEmail(user.getEmail())
-        .stream()
-        .map(Dataset::getDataSetId)
-        .collect(Collectors.toList());
+  /**
+   * Find all dataset ids by the DAC User. Will return ids for Chairpersons or Members
+   * @param user The DAC User
+   * @return List of Dataset IDs
+   */
+  public List<Integer> findDatasetIdsByDACUser(User user) {
+    return datasetDAO.findDatasetIdsByDACUserId(user.getUserId());
```

Comment:
why change from user email to user id?

Reply:
Yes, typically you want to query by the integer primary key because it will be indexed, which makes for fast searches.